content/user/ui/report/Vulnerability-trends.md

```diff
@@ -0,0 +1,71 @@
+<!--
+title: "Vulnerability Trends"
+description: "Overview of vulnerability trend reports"
+tags: "TeamServer application vulnerability PDF report"
+-->
+ 
+Vulnerability management is a vital responsibility of any security team. Recognizing the vulnerabilities your applications face and how well they're being managed can help you understand your security posture.
+
+The report presents data about newly discovered vulnerabilities as well as accumulated total vulnerabilities in comparison to remediations. Focus on the details that you care about by using robust filtering options to look at data over any applications, servers, Assess rules or by vulnerability severity. Contrast can also present data in various timeframes, ranging from the last week to the last year. The customized results allow you to prioritize efforts toward vulnerability remediation.
+
+## Access Data
+ 
+Select **Reports** in the User menu to go the Vulnerability Trend dashboard. Click the **View** link to see the graphs in more detail. 
+
+Select **New** to see a graph of new vulnerabilities. Select **Total** to see a graph of all reported vulnerabilities compared to all remediated vulnerabilities. Each point in the graph generates a tooltip with the number of suspicious, confirmed and reported vulnerabilities on that date when you hover over it. 
```

Comment:
When you reference the breakdowns shown in the tooltip, you only mention the statuses of unremediated vulns. The green data points show the breakdown of remediated vulns, which includes Not A Problem, Remediated and Fixed. So, just wondering if we should include that here somewhere. Could be referenced separately or together, i.e. "Each data point generates a tooltip on hover including a breakdown of status: Suspicious, Confirmed, Reported, Not A Problem, Remediated or Fixed."